### DISCUSSION TITLE
[Custom]: how i can stop vscode theme change when i change hyprdots theme

### DISCUSSION THREAD

**Discussion by nuexq** [prasanthrangan/hyprdots#1090] [Q&A]

**Comment by kRHYME7** [comment]
2 ways. 

remove the extensions  assigned on each theme. The control file is ``` ~/.config/hypr/theme.ctl ```
+ Imagine it as columns separated by "|". 
+ Remove everything on the 3rd column as it is the vs code extension. 
+ it should look like "||" remember do not remove/add any "|"

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/1ea0f357-a10c-4c58-9e17-e12d020dffbf)


Or remove the line that changes it on the script
``` ~/.config/hyprdots/scripts/themeswitch.sh```

you can comment out this lines 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/f29984e6-5707-4041-9327-079f0ef6548d)

**Reply by nuexq** [reply]
thanks for help that's work, i want to ask how to remove blender and other apps (it's no installed by pacman or paru)

**Comment by kRHYME7** [comment]
I'll be moving this to discussions panel.
